Getting started: community (#76)
codewars/docs: content/getting-started/community.md

# Community

> Codewars gathers thousands of users of many skills and levels of experience. Their knowledge is invaluable resource and when used correctly, your training can be easier, and extremely effective.

Reviewer: ```suggestion
Codewars gathers thousands of users with many different skills and levels of experience. Their knowledge is an invaluable resource and they can make your training easier, and extremely effective.
```

As Dan said in gitter about the "trouble shooting" document, starting with something a bit negative (like "when used correctly") might not be welcoming (even if true). In addition, "using" people leaves me a bad feeling when I read it. ;o

Author: I don't get the part about "used correctly" being negative. Does it sound so?

About "using" part, it's about using the knowledge, and not about using people. It's quite common form in my language, does it indeed sound wrong in English?

Reviewer 2: > Codewars gathers thousands of users

Not sure if we should mention the size of the community. But this is inaccurate since we have over 1.7M users total and at least 400K active users in the past 6 months.

## Kata Discourse

> If you want to discuss some particular kata, you can use its discourse page to do so. It's a good place to ask for explanation, search for help with solution, add your opinion on it, or report problems. You can label your post with one of possible labels:

Reviewer: ```suggestion
If you want to discuss some particular kata, you can use its discourse page to do so.

It's a good place to ask for explanations, search for help about solution, add your opinion on it, or report problems. You can label your post with one of possible labels when appropriate:
```

- **Question** - if description is not clear, you need help, got stuck, or you cannot get your solution to work,
- **Suggestion** - if you have an idea how author could change the kata, its description, or tests, to make it better,
- **Issue** - if you think the kata is broken, and author needs to fix it in some way.

> 

Reviewer: ```suggestion

See this [detailed documentation (TODO: direct link to Labels reference)]() about the different kinds of labels and their use.

```

Author: Since posting bogus issues is one of the most common beginner mistakes, I think it will be good to have labels briefly explained here (with a link to actual, full definition)

Reviewer: yes, my suggestion is just the addition of that link (with emphasis about its existence).

You should also mark your posts with **Spoiler** flag if they contain some code (even not fully working one), or reveal some significant details of the solution.

## Communication

Remember that your posts will be read by other people, and to be able to help you or answer your questions, they have to be able to understand you properly. Do all you can to express yourself clearly, describe your point with all necessary details, use appropriate labels and code formatting when needed.

There's a set of rules you should follow to keep interaction with other users a good experience for both sides. You can find them in _(TODO: where? Code of Conduct, some discourse guidelines?)_.

## Other Places

Kata discourse is not the only place to meet other users. Community members can be also met at:

- Codewars [Gitter](https://gitter.im/Codewars/codewars.com) with a dedicated room for [kata solving help](https://gitter.im/Codewars/codewars.com/kata-solving-help),

> - [Chat (TODO: link when ready)]() on Zulip,
> - Codewars [forum (TODO: link to Discourse forum)]()

Reviewer 2: ```suggestion
<!--
- [Chat (TODO: link when ready)]() on Zulip,
- Codewars [forum (TODO: link to Discourse forum)]()
-->
```
